packages/node_modules/@webex/internal-plugin-calendar/src/index.js

```diff
@@ -82,6 +103,44 @@ registerInternalPlugin('calendar', Calendar, {
             ctx.transform('decryptTextProp', 'spaceURL', object.encryptionKeyUrl, object)
           ].concat(decryptedParticipants));
         }
+      },
+      {
+        name: 'transformMeetingNotes',
+        direction: 'inbound',
+        fn(ctx, object) {
+          if (!object) {
+            return Promise.resolve();
+          }
+
+          if (!object.encryptionKeyUrl) {
+            return Promise.resolve();
+          }
+
+          return Promise.all([
+            ctx.transform('decryptTextProp', 'encryptedNotes', object.encryptionKeyUrl, object)
+          ]);
+        }
+      },
+      {
+        name: 'transformMeetingParticipants',
+        direction: 'inbound',
+        fn(ctx, object) {
+          if (!object) {
+            return Promise.resolve();
+          }
+
+          if (!object.encryptionKeyUrl) {
```

Comment:
Do we need to add a check for the `encryptedParticipants` object here? That way we won't get an error if it doesn't exist and we try to `.map` it on line 137.

```suggestion
          if (!object.encryptionKeyUrl || !object.encryptedParticipants) {
```

Reply:
Will do